(Nextjs quickstart) Tweak middleware bulletpoints to clarify use of `src` (#999)
clerk/clerk-docs: docs/quickstarts/nextjs.mdx

[`clerkMiddleware()`](/docs/references/nextjs/clerk-middleware#clerk-middleware) grants you access to user authentication state throughout your application, on any route or page. It also allows you to protect specific routes from unauthenticated users. To add `clerkMiddleware()` to your application, follow these steps:

1. Create a `middleware.ts` file.

>     - If you are using the `/src` directory, place `middleware.ts` in the `/src` directory.
>     - If you are not using the `/src` directory, place `middleware.ts` in the root directory alongside `.env.local`.

Reviewer: ```suggestion
    - If you're using the `/src` directory, place `middleware.ts` in the `/src` directory.
    - If you're not using the `/src` directory, place `middleware.ts` in the root directory alongside `.env.local`.

```
